ios/FluentUI/Card Nudge/CardNudgeView.swift

```diff
@@ -0,0 +1,195 @@
+    ///
+    /// To show an action button, provide values for both `actionButtonTitle` and  `actionButtonAction`.
+    @Published @objc public var actionButtonAction: CardNudgeViewButtonAction?
+
+    /// Action to be dispatched by the dismiss ("close") button on the trailing edge of the control.
+    @Published @objc public var dismissButtonAction: CardNudgeViewButtonAction?
+
+    /// Parent window of the `PersonaButton`.
+    ///
+    /// Used to derive the color theme for the control.
+    var hostingWindow: UIWindow? {
+        didSet {
+            tokens.window = hostingWindow
+        }
+    }
+
+    let tokens: CardNudgeTokens
+
+    @objc init(style: CardNudgeViewStyle, title: String) {
+        self.style = style
+        self.title = title
+        self.tokens = CardNudgeTokens(style: style)
+
+        super.init()
+    }
+
+    @objc convenience init(style: CardNudgeViewStyle,
+                           title: String,
+                           subtitle: String? = nil,
+                           mainIcon: UIImage? = nil,
+                           accentText: String? = nil,
+                           actionButtonTitle: String? = nil,
+                           actionButtonAction: CardNudgeViewButtonAction? = nil,
+                           dismissButtonAction: CardNudgeViewButtonAction? = nil) {
+        self.init(style: style, title: title)
+
+        self.subtitle = subtitle
+        self.mainIcon = mainIcon
+        self.accentText = accentText
+        self.actionButtonTitle = actionButtonTitle
+        self.actionButtonAction = actionButtonAction
+        self.dismissButtonAction = dismissButtonAction
+    }
+}
+
+public struct CardNudgeView: View {
+    @ObservedObject var tokens: CardNudgeTokens
+    @ObservedObject var state: CardNudgeViewState
+
+    @ViewBuilder
+    var icon: some View {
+        if let icon = state.mainIcon {
+            ZStack {
+                Circle()
+                    .frame(width: tokens.circleSize, height: tokens.circleSize)
+                    .foregroundColor(Color(tokens.buttonBackgroundColor))
+                Image(uiImage: icon)
+                    .renderingMode(.template)
+                    .frame(width: tokens.iconSize, height: tokens.iconSize, alignment: .center)
+                    .foregroundColor(Color(tokens.accentColor))
+            }
+            .padding(.trailing, tokens.innerPadding)
+        }
+    }
+
+    private var hasSecondTextRow: Bool {
+        state.accentIcon != nil || state.accentText != nil || state.subtitle != nil
+    }
+
+    @ViewBuilder
+    var textContainer: some View {
+        VStack(alignment: .leading, spacing: tokens.interTextVerticalPadding) {
+            Text(state.title)
+                .font(.subheadline)
+                .fontWeight(.medium)
+                .lineLimit(1)
+                .foregroundColor(Color(tokens.textColor))
+
+            if hasSecondTextRow {
+                HStack(spacing: tokens.accentPadding) {
```

Comment:
Do you think the text items need padding as well? This is how the card looks like with all the settings enabled (The subtitle gets cut off right by the action button): 
![Screen Shot 2021-07-27 at 10 50 19 AM](https://user-images.githubusercontent.com/22566866/127203619-07cf3877-2621-426d-a677-b66b2f355a9f.png)

Reply:
Ah that's not great. Yes, I think a little right padding would be better.